.github/workflows/docs.yml

```
@@ -4,12 +4,12 @@ permissions: read-all
 
 on:
   push:
-    branches: [main, develop, feature/*, alpha, beta, release/*]
+    branches: [ main, develop, feature/*, alpha, beta, release/* ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)